### PR TITLE
Add SDK specific user agent string

### PR DESCRIPTION
**Pull request by nilsgaeq**
Add a User-Agent string for the SDK, to enable extraction of usage statistics from http request logs.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Not tested

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Environment:

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding changes to the architecture
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: omnia_timeseries/http_client.py [hunk 1]
```diff
@@ -5,17 +5,22 @@
...
-    def __init__(self, response:Response) -> None:
+    def __init__(self, response: Response) -> None:
         error = json.loads(response.text)
         self._status_code = response.status_code
         self._reason = response.reason
         self._message = error["message"]
         self._trace_id = error["traceId"]
-        super().__init__(f"Status code: {self._status_code}, Reason: {self._reason}, Message: {self._message},  Trace ID: {self._trace_id}")
+        super().__init__(
+            f"Status code: {self._status_code}, Reason: {self._reason}, Message: {self._message},  Trace ID: {self._trace_id}")
```
Comment (eaidland) [inline omnia_timeseries/http_client.py:23]: Why space?

**Review by eaidland — APPROVED** [review]

File: omnia_timeseries/http_client.py (hunk 1 above)
Comment (nilsgaeq) [reply omnia_timeseries/http_client.py:23]: Which one? This is just reformatted code, according to python conventions. Any whitespace in the string is just at is has been.

**Review by runaraj — APPROVED** [review]

File: omnia_timeseries/http_client.py (hunk 1 above)
Comment (nilsgaeq) [reply omnia_timeseries/http_client.py:23]: @eaidland?

Comment (eaidland) [reply omnia_timeseries/http_client.py:23]: Ok it's prob just the formatter

Comment (nilsgaeq) [reply omnia_timeseries/http_client.py]: There was an extra space before `Trace ID` though, so removed it

**Pull request merged**